Drv/TcpClient/test/ut/TcpClientTester.cpp

```diff
@@ -180,7 +181,7 @@ void TcpClientTester ::test_no_automatic_recv_connection() {
 }
 
 // ----------------------------------------------------------------------
-// Handlers for typed from ports
+// Handler overrides for typed from ports
```

Comment:
handlers not overridden here (including `dataReturnOut`) are autocoded with `UT_AUTO_HELPERS`